### ISSUE TITLE
QWebView Crashes inside a dll 

### ISSUE THREAD

**Issue by dexter4life** [qtwebkit/qtwebkit#999]
Something as basic as 

```
class SOME_EXPORT_API MyClass{
public:
  void someOtherFunctions();
private:
QWebView m_view;
}
```

When compiled and the dll is loaded, even without making any form of reference to the dll directly it will crash without showing any warning or something simple to understand.

I suspected that the constructor is called in that declaration as done but what is the reason behind this? how can I remedy this situation to make it work at least to show

Does the QWebView need to be a child of a QWidgets to be come accessible?

**Comment by annulen** [comment]
Because it happens on load, I guess it's caused by missing dependencies of Qt5WebKit.dll. If you are using binaries published on GitHub release, they contain 5 non-Qt dll files which are required to be present in searchable path when you are running your application.

**Comment by dexter4life** [comment]
Yes, I know, but it does happen even in Static build of QWebView which is what I used currently and experience the crashing in most cases.

**Comment by annulen** [comment]
Can you provide backtrace from crash?

**Comment by dexter4life** [comment]
i will try that.
